application/common/models/Reset.php

```diff
@@ -331,12 +331,13 @@ private function sendEmail($toAddress, $subject, $view, $ccAddress = null)
     /**
      * Check if user provided code is valid
```

Comment:
Does this function actually check whether the code is valid (which sounds like it would require it to both be correct and be unexpired) or does it just check whether the code is correct (as the function name itself suggests)?

Reply:
Given the way we're using this method, it looks like it should just check whether the code is correct, since we need to be able to do that separately from checking whether the code is expired (in order to only restart expire password resets if a valid code was provided).

Reply:
Maybe I'm not understanding correctly, but the method does as the name suggests...checks whether the code matches. It also tracks attempts, but that's unrelated to expiration.